overlord/snapstate/autorefresh.go

```diff
@@ -60,6 +60,14 @@ var (
 	CanAutoRefresh        func(st *state.State) (bool, error)
 	CanManageRefreshes    func(st *state.State) bool
 	IsOnMeteredConnection func() (bool, error)
+
+	defaultRefreshSchedule = func() []*timeutil.Schedule {
+		refreshSchedule, err := timeutil.ParseSchedule(defaultRefreshScheduleStr)
+		if err != nil {
+			panic(fmt.Sprintf("defaultRefreshSchedule cannot be parsed: %s", err))
```

Comment:
Should this be "`%v`"?

Reply:
It shouldn't make a difference since both cause `Error()` to be called and printed